### ISSUE TITLE
"ResidentialSales.dbf" file is missing,

### ISSUE THREAD

**Issue by srahman16** [PacktPublishing/Spatial-Analytics-with-ArcGIS#1]
"**ResidentialSales.dbf**" file is missing,
![image](https://user-images.githubusercontent.com/17202080/181913752-d00c9f1f-13e5-492d-b87a-cba59dfd870e.png)

**Comment by 29noL** [comment]
How to workaround that:
Open QGIS and drag the ResidentialSales.shp file into the map. (QGIS is more forgiving and will often render the geometry even without the database file).

Right-click the layer in the QGIS Layers panel -> Export -> Save Features As...

Choose ESRI Shapefile as the format and save it as a new file (e.g., ResidentialSales_Fixed.shp).

In ArcMap, add this new fixed shapefile. It will now load perfectly because QGIS generated a brand-new, indexed .dbf that matches the point count of the geometry.

Note: The attribute table will be empty (or only contain IDs), but for the Chapter 4 exercises you only need the point locations, so this fix works perfectly to complete the lesson!


TLDR (Too Long Didn't Read):
the missing .dbf doesn't allow the shapefile ResidentialSales from being opened in ArcMap but in QGIS you can open ResidentialSales.shp just fine and export it thus replacing the missing .dbf file (allowing you to continue with the exercise

[ResidentialSales.zip](https://github.com/user-attachments/files/26535923/ResidentialSales.zip)


Godspeed!
